shaka/src/memory/object_tracker.cc

```diff
@@ -53,15 +53,17 @@ void ObjectTracker::RemoveRef(const Traceable* object) {
   if (object) {
     std::unique_lock<Mutex> lock(mutex_);
     auto* key = const_cast<Traceable*>(object);  // NOLINT
-    DCHECK_EQ(objects_.count(key), 1u);
-    CHECK_GT(objects_[key], 0u);
-    objects_[key]--;
+    auto ref_count = objects_.find(key);
+    DCHECK(ref_count != objects_.end());
+    CHECK_GT(ref_count, 0u);
+    ref_count--;
```

Comment:
The iterator points to a `std::pair`.  This also moves the iterator, not change the value.

```c++
CHECK_GT(ref_count->second, 0u);
ref_count->second--;
```